src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -5688,6 +5688,15 @@ std::string SurgeGUIEditor::modulatorName( int i, bool button )
             sprintf( txt, "%s MSEG %d", (isS ? "Scene" : "Voice" ), fnum + 1 );
          return std::string( txt );
       }
+      else if( lfodata->shape.val.i == ls_function)
+      {
+         char txt[64];
+         if( button )
+            sprintf( txt, "%sFUN %d", (isS ? "S-" : "" ), fnum + 1 );
```

Comment:
So I'm fine having this button be active but I think the UI should show what it's doing so my only change is that this should be ENV / ENVELOPE for now.

Reply:
The idea was to create hype for function/formula modulator :) Maybe we get a dev or two interested enough, haha!

Reply:
Oh also the reason I renamed it to Function is because FUN is a nicer shorthand than FORM or FML. 😁